src/internal/m365/collection/exchange/collection.go

```diff
@@ -299,6 +310,12 @@ func (col *prefetchCollection) streamItems(
 
 			stream <- item
 
+			col.Counter.Add(count.StreamBytesAdded, info.Size)
+
+			if col.Counter.IncRead(count.StreamItemsAdded)%1000 == 0 {
+				logger.Ctx(ctx).Infow("item stream progress", "stats", col.Counter.Values())
```

Comment:
Could you modify this log a bit so that we can distinguish item removals from additions?

Reply:
can do!